File: Java/Tests/jdi-uitest-wintests/src/test/java/com/epam/jdi/uitests/testing/unittests/tests/composite/FrameTests.java
package com.epam.jdi.uitests.testing.unittests.tests.composite;

import com.epam.jdi.uitests.testing.unittests.W3CInit;
import com.epam.web.matcher.testng.Assert;
import org.testng.annotations.Test;

import static com.epam.jdi.uitests.testing.unittests.pageobjects.w3cSite.W3cSite.framePage;

/**
 * Created by Dmitry_Lebedev1 on 10/15/2015.
 */
public class FrameTests extends W3CInit {
    @Test(enabled = false)
    public void fillTest() {
        framePage.tryIt.click();
        Assert.areEquals(() -> framePage.frame.label.getText(), "Click Me!");
    }

}

File: Python/Test/jdi_uitests_webtests/test/composite/frame_tests.py
import time

from JDI.jdi_assert.testing.assertion import Assert
from Test.jdi_uitests_webtests.main.page_objects.w3c_site.w3c_site import W3cSite
from Test.jdi_uitests_webtests.test.init_tests import W3CInit


class FrameTests(W3CInit):

    @staticmethod
    def test_fill():
        W3cSite.frame_page.try_it_button.click()
        Assert.assert_equal(W3cSite.frame_page.i_frame.label_button.get_text(), "Click Me!")
